swarm_copy/app/dependencies.py

```diff
@@ -5,18 +5,43 @@
 from typing import Annotated, Any, AsyncIterator
 
 from fastapi import Depends, HTTPException, Request
+from fastapi.security import HTTPBearer
+from httpx import AsyncClient, HTTPStatusError
+from keycloak import KeycloakOpenID
 from openai import AsyncOpenAI
 from sqlalchemy.ext.asyncio import AsyncEngine, AsyncSession
+from starlette.status import HTTP_401_UNAUTHORIZED
 
 from swarm_copy.app.config import Settings
 from swarm_copy.app.database.sql_schemas import Threads
+from swarm_copy.cell_types import CellTypesMeta
 from swarm_copy.new_types import Agent
 from swarm_copy.run import AgentsRoutine
-from swarm_copy.tools import PrintAccountDetailsTool
+from swarm_copy.tools.electrophys_tool import ElectrophysTool
+from swarm_copy.tools.get_me_model_tool import GetMEModelTool
+from swarm_copy.tools.get_morpho_tool import GetMorphoTool
+from swarm_copy.tools.kg_morpho_features_tool import KGMorphoFeatureTool
+from swarm_copy.tools.literature_search_tool import LiteratureSearchTool
+from swarm_copy.tools.morphology_features_tool import MorphologyFeatureTool
+from swarm_copy.tools.resolve_entities_tool import ResolveEntitiesTool
+from swarm_copy.tools.traces_tool import GetTracesTool
```

Comment:
These should be part of the `__init__.py` as we do in `neuroagent`, so that we don't have to import them from their respective file.